### PR TITLE
[W/A] disable group conv in asm igemm solver

### PR DESCRIPTION
**Pull request by junliume**
disable group conv in asm igemm solver.

Workaround for Issue #1979 

P.S. removal of WORKAROUND_ISSUE_1053 as a by-product (6 months after WA is removed, eliminate related code)

### CODE REVIEW

**Review by carlushuang — DISMISSED** [review]
LGTM

**Review by carlushuang — DISMISSED** [review]
LGTM

**Review by atamazov — CHANGES_REQUESTED** [review]
- PR description is missing the reasoning
- W/A does not comply #1685 

Please do not hesitate to ask for assistance, if you need it (or if do not have time).

**Comment by junliume** [comment]
@carlushuang  @atamazov : updated this PR to comply with [MIOpen lifecycle of workarounds](https://github.com/ROCmSoftwarePlatform/MIOpen/discussions/1685)

File: src/include/miopen/conv/asm_implicit_gemm.hpp [hunk 1]
```diff
@@ -34,6 +34,10 @@
 #include <vector>
 #include <limits>
 
+/// W/A for issue 1979: igemm solver does not support group conv. See:
+/// https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1979
```
Comment (atamazov) [inline src/include/miopen/conv/asm_implicit_gemm.hpp:38]: [Notice] Basically, `WORKAROUND_ISSUE_1979` says everything so the comment is not required.

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -48,7 +48,6 @@ set_var_to_condition(MIOPEN_TEST_WITH_MIOPENDRIVER_DEFAULT MIOPEN_BUILD_DRIVER)
 option( MIOPEN_TEST_WITH_MIOPENDRIVER "Use MIOpenDriver in tests" ${MIOPEN_TEST_WITH_MIOPENDRIVER_DEFAULT})
 
 option( WORKAROUND_ISSUE_936 "" ON)
-option( WORKAROUND_ISSUE_1053 "" OFF) # TODO: Remove this W/A after ~6 months (in January 2023)
```
Comment (atamazov) [inline test/CMakeLists.txt:51]: [Notice] The removal of `WORKAROUND_ISSUE_1053` is a by-product which is not mentioned in the description.

**Review by atamazov — APPROVED** [review]
LGTM!

**Review by carlushuang — APPROVED** [review]
LGTM

**Pull request merged**